defaults/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Role variable defaults and documentation.
+
+# This is a list of the settings to apply using `sysctl`. Each list item is a
+# `dict` with allowed key names of `name`, `value`, `state`, and `previous`.
+# The settings are specified with a `name`, a `value`, and an optional `state`
+# or `previous` which controls the behavior.  For example:
+# kernel_settings_sysctl:
+#   - name: fs.epoll.max_user_watches
+#     value: 785592
+#   - name: fs.file-max
+#     value: 379724
+kernel_settings_sysctl: null
```

Comment:
Wouldn't it be more natural to use an empty list as defaults for all those settings? Those settings are essentially lists of things to be set on the target system. By default there is nothing to be set if the user does not specify it -> this is expressed by an empty list.

Reply:
thanks - https://github.com/linux-system-roles/kernel_settings/pull/1/commits/b7cccb8db5289d08b8d9b03ca749abc10e5308d1